Dockerfile

```
@@ -0,0 +1,47 @@
+# Docker image for running Frequency parachain node container (with collating)
+# locally in instant seal mode then deploying schemas to that node.
+
+#This pulls the latest instant-seal-node image
+FROM frequencychain/instant-seal-node as frequency-image
+
+#Switch to root to install node on image
+USER root
+
+LABEL maintainer="Frequency"
+LABEL description="Frequency collator node in instant seal mode with schemas deployed"
+
+RUN apt-get update && apt-get install -y ca-certificates && update-ca-certificates
+
+# Install node-js to base image
+RUN apt-get update && apt-get install -y curl gnupg
+RUN curl -sL https://deb.nodesource.com/setup_16.x | bash
+RUN apt-get update && apt-get install -y nodejs
```

Not blocking, but you can get a smaller image by combining these along with an apt clean into one line.

Yes it is a harder to read, but each RUN command will build a separate layer. Also these really should never change without all of them changing.

See docs: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

```suggestion
RUN apt-get update && apt-get install -y curl gnupg \
        && curl -sL https://deb.nodesource.com/setup_16.x | bash \
        && apt-get update && apt-get install -y nodejs \
        && rm -rf /var/lib/apt/lists/*
```